cnf-certification-test/tnf-config/tnf_config.yml

```diff
@@ -0,0 +1,11 @@
+targetNameSpaces:
+    - name: observability-ns
+targetPodLabels:
+    - prefix: tnf-qe
+      name: observability
```

Comment:
leftover ?

Reply:
@edcdavid [The Leftovers](https://en.wikipedia.org/wiki/The_Leftovers_(TV_series)) :-)

Reply:
ahah didn't know this show, might even check it out :-)

Reply:
Yeah, check it out, I recommend, the music theme is intense. There are no superstars, but I adore Amy Brenneman (The Heat). I'm planning to rewatch all seasons.

Reply:
Removed.